src/test/java/net/sf/jabref/model/entry/BibEntryTests.java

```diff
@@ -212,16 +212,21 @@ public void setNullField() {
 
     @Test(expected = NullPointerException.class)
     public void addNullKeywordThrowsNPE() {
-        keywordEntry.addKeyword(null);
+        keywordEntry.addKeyword(null, Globals.prefs.get(JabRefPreferences.KEYWORD_SEPARATOR));
```

Comment:
Does that really make sense? I just had to add some stuff in `LatexFieldFormatterPreferences` as some other behaviour really relies on the preferences but it is not handled in the tests.